gammapy/image/maps.py

```diff
@@ -499,6 +500,7 @@ def write(self, filename=None, header=None, **kwargs):
         hdulist = fits.HDUList()
         for name in self.get('_map_names', sorted(self)):
             hdu = self[name].to_image_hdu()
+            hdu.name = name
```

Comment:
This is really not important, but if you have time to find out, I'd be interested in why this error happens...because it might be a bug in `.to_image_hdu()` or somewhere else.